### ISSUE TITLE
Could not find active_link_to-1.0.5 in any of the sources

### ISSUE THREAD

**Issue by williamhatch** [comfy/comfortable-mexican-sofa#851]
the steps:
1. rails new mark -d postgresql
2. add comfortable-mexican-sofa to gem
3. bundle
4. rails generate comfy:cms

then show the error:
Could not find active_link_to-1.0.5 in any of the sources

and rails about:
About your application's environment
Rails version             5.2.1
Ruby version              2.5.1-p57 (x86_64-darwin17)
RubyGems version          2.7.7
Rack version              2.0.5
JavaScript Runtime        Node.js (V8)
Middleware                Rack::Sendfile, ActionDispatch::Static, ActionDispatch::Executor, ActiveSupport::Cache::Strategy::LocalCache::Middleware, Rack::Runtime, Rack::MethodOverride, ActionDispatch::RequestId, ActionDispatch::RemoteIp, Sprockets::Rails::QuietAssets, Rails::Rack::Logger, ActionDispatch::ShowExceptions, WebConsole::Middleware, ActionDispatch::DebugExceptions, ActionDispatch::Reloader, ActionDispatch::Callbacks, ActiveRecord::Migration::CheckPending, ActionDispatch::Cookies, ActionDispatch::Session::CookieStore, ActionDispatch::Flash, ActionDispatch::ContentSecurityPolicy::Middleware, Rack::Head, Rack::ConditionalGet, Rack::ETag, Rack::TempfileReaper
Application root          /Users/william/tmp/mark
Environment               development
Database adapter          postgresql
Database schema version   0

and BUNDLE_PATH=vendor/cache

what is the problem

**Comment by GBH** [comment]
It's the same issue as https://github.com/comfy/active_link_to/issues/59#issuecomment-436004033

Did you try stackoverflow as this sounds like a app or system configuration issue?

**Comment by williamhatch** [comment]
thanks, GBH.  
I havent tried stackoverflow yet. I unset BUNDLE_PATH and it works... might be something is hardcoded? I am not quite sure but it is very weird.

BTW, to build a CMS, what is your suggestion:
1. from comfortable-mexican-sofa directly
2. from comfy_blog github
3. from comfy_demo github

just a little confusing.

**Comment by GBH** [comment]
You add CMS to a Rails app. This CMS is not a stand-alone. You create a new project `rails new your_project_name`. Then add `gem "comfortable_mexican_sofa", "~> 2.0.0"` to the `Gemfile`. Follow the install steps: https://github.com/comfy/comfortable-mexican-sofa#installation

**Comment by williamhatch** [comment]
thanks for the quick response @GBH .  I tried to seek help from gitter or twitter as info shown below: 
Gitter: https://gitter.im/comfy/comfortable-mexican-sofa Twitter: @GroceryBagHead

I would like to dig more for rails cms, try to find/build one competitive with php craftcms ... LOL. Possible to have a channel like slack/mail communication etc. It would be more efficient than here :)

BTW, I just finished the blog demo by adding comfy_blog to Gemfile... nice!
